.gitignore

```diff
@@ -1,4 +1,5 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+.env.local
```

Comment:
If this file is required for developers sometimes it's nice to add a .env.sample file to the project and outline which env variables a dev needs to have set up for the project. Then, in the readme you can tell them to `cp .env.sample .env.local` and fill in the variables as needed. 

```
.env.sample
NODE_MAILER_EMAIL=development@spec.org // this can probably be hardcoded as it's not a password
NODE_MAILER_PASSWORD=< reach out to dev for password >
```

Reply:
@mjtribble That is really helpful feedback. Thank you!